administrator/components/com_content/views/shared/view.html.php

```diff
@@ -0,0 +1,142 @@
+		$options[] = JHtml::_('select.option', '2', JText::_('J2'));
+		$options[] = JHtml::_('select.option', '3', JText::_('J3'));
+		$options[] = JHtml::_('select.option', '4', JText::_('J4'));
+		$options[] = JHtml::_('select.option', '5', JText::_('J5'));
+		$options[] = JHtml::_('select.option', '6', JText::_('J6'));
+		$options[] = JHtml::_('select.option', '7', JText::_('J7'));
+		$options[] = JHtml::_('select.option', '8', JText::_('J8'));
+		$options[] = JHtml::_('select.option', '9', JText::_('J9'));
+		$options[] = JHtml::_('select.option', '10', JText::_('J10'));
+		$this->f_levels = $options;
+		$this->addToolbar();
+		$this->sidebar = JHtmlSidebar::render();
+		parent::display($tpl);
+	}
+	/**
+	 * Add the page title and toolbar.
+	 *
+	 * @return  void
+	 *
+	 * @since   1.6
+	 */
+	protected function addToolbar()
+	{
+		$state = $this->get('State');
+		$canDo = JHelperContent::getActions('com_content', 'category', $this->state->get('filter.category_id'));
+		JToolbarHelper::title(JText::_('COM_CONTENT_SHARED_TITLE'), 'star featured');
+		if ($canDo->get('core.create'))
+		{
+			JToolbarHelper::addNew('article.add');
+		}
+		if ($canDo->get('core.edit'))
+		{
+			JToolbarHelper::editList('article.edit');
+		}
+		if ($canDo->get('core.edit.state'))
+		{
+			JToolbarHelper::publish('articles.publish', 'JTOOLBAR_PUBLISH', true);
+			JToolbarHelper::unpublish('articles.unpublish', 'JTOOLBAR_UNPUBLISH', true);
+			JToolbarHelper::custom('articles.unfeatured', 'unfeatured.png', 'featured_f2.png', 'JUNFEATURE', true);
+			JToolbarHelper::archiveList('articles.archive');
+			JToolbarHelper::checkin('articles.checkin');
+		}
+		if ($state->get('filter.published') == -2 && $canDo->get('core.delete'))
+		{
+			JToolbarHelper::deleteList('JGLOBAL_CONFIRM_DELETE', 'articles.delete', 'JTOOLBAR_EMPTY_TRASH');
+		}
+		elseif ($canDo->get('core.edit.state'))
+		{
+			JToolbarHelper::trash('articles.trash');
+		}
+		if ($canDo->get('core.admin') || $canDo->get('core.options'))
+		{
+			JToolbarHelper::preferences('com_content');
+		}
+		JToolbarHelper::help('JHELP_CONTENT_SHARED_ARTICLES');
+	}
+	/**
+	 * Returns an array of fields the table can be sorted by
+	 *
+	 * @return  array  Array containing the field name to sort by as the key and display text as value
+	 *
+	 * @since   3.0
+	 */
+	protected function getSortFields()
+	{
+		return array(
+			'fp.ordering' => JText::_('JGRID_HEADING_ORDERING'),
+			'a.state' => JText::_('JSTATUS'),
+			'a.title' => JText::_('JGLOBAL_TITLE'),
+			'category_title' => JText::_('JCATEGORY'),
+			'access_level' => JText::_('JGRID_HEADING_ACCESS'),
+			'a.created_by' => JText::_('JAUTHOR'),
+			'language' => JText::_('JGRID_HEADING_LANGUAGE'),
+			'a.created' => JText::_('JDATE'),
+			'a.id' => JText::_('JGRID_HEADING_ID')
+		);
+	}
+	/**
+	 * Form method
+	 *
+	 * @return  form object
+	 *
+	 * @since   _DEPLOY_VERSION_
+	 */
+	 public function getForm()

```

Comment:
JForm::getInstance could not load file.....got this error, solution to this? @roland-d
Will this method suppose to help in that?